server/routes/bookedtoday/arrivals/courtreturns/checkCourtReturnController.test.ts

```diff
@@ -52,6 +53,20 @@ describe('checkCourtReturnController', () => {
         .expect(res => {
           const $ = cheerio.load(res.text)
           expect($('h1').text()).toContain('This person is returning from court')
+          expect($('.data-qa-per-record-prisoner-name').text()).toContain(`${arrival.firstName} ${arrival.lastName}`)
```

Comment:
Didn't put this in the int tests since it's tested here. Thought about moving it into the commonTests.ts file, but got a bit bogged down with the typing 😆

Reply:
This stuff should really go in an int test - don't really want these tests super coupled tightly to the view which is why we generally only check the title.

I've been stripping out this from other controller tests